src/main/scala/com/fulcrumgenomics/fastq/FastqToBam.scala

```diff
@@ -92,6 +92,7 @@ class FastqToBam
   @arg(flag='s', doc="If true, queryname sort the BAM file, otherwise preserve input order.")  val sort: Boolean = false,
   @arg(flag='u', doc="Tag in which to store molecular barcodes/UMIs.")                         val umiTag: String = ConsensusTags.UmiBases,
   @arg(flag='q', doc="Tag in which to store molecular barcode/UMI qualities.")                 val umiQualTag: Option[String] = None,
+  @arg(doc="Store the sample barcode qualities in the QT Tag.")                                val storeSampleBarcodeQualities: Boolean = false,
```

Comment:
Why there is no `flag` ? What will the option char for enabling `QT` ?

Reply:
Use the full command line option `--store-sample-barcode-qualities`.  I don't think there's a flag that makes sense, and likely since folks are implementing this in a workflow software, I don't think it's too onerous.  If you really want one, I'd pick `x` since it's sufficiently different than the others.

Reply:
I'd probably just add `-Q` if it were up to me